docs/x509/reference.rst

```diff
@@ -432,9 +432,6 @@ X.509 Certificate Object
         :raises cryptography.x509.UnsupportedGeneralNameType: If an extension
             contains a general name that is not supported.
 
-        :raises UnicodeError: If an extension contains IDNA encoding that is
```

Comment:
This has been untrue since 2.1 and we never noticed. Since we don't compute U-labels we won't error.